application/models/TemplateConfiguration.php

```diff
@@ -492,11 +493,11 @@ public function searchGrid()
         $criteria->addCondition('t.sid IS NULL');
         $criteria->addCondition('template.name IS NOT NULL');
 
-        // check if survey group id is present
-        $gsid = App()->request->getQuery('id', null);
         if ($gsid !== null) {
-            $criteria->addCondition('t.gsid = ' . $gsid);
+            /* Group configuration */
+            $criteria->compare('t.gsid', $gsid);
```

Comment:
Please correct me if I'm wrong, here we just comparing? Do we need any additional action here?

Reply:
What do you mean ? 

compare do 

- condition = "t.gsid = :gsid"
- params [":gsid" => $gsid)

https://github.com/yiisoft/yii/blob/8530c30959d5f68c0d1b0d611e3ce2bf32a8371a/framework/db/schema/CDbCriteria.php#L449-L450

Reply:
ok, got it, thank you